### DISCUSSION TITLE
Welcome to react-native-hold-menu Discussions!

### DISCUSSION THREAD

**Discussion by enesozturk** [enesozturk/react-native-hold-menu#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by stefanleoussis** [comment]
Does this package work for Android?

**Comment by ameer-taghavi** [comment]
Yes, it works but has too lag and poor performance. #60

**Reply by stefanleoussis** [reply]
I see... Thank you

**Reply by enesozturk** [reply]
Yeah, it has issues and is not production-ready on Android yet.

**Reply by fukemy** [reply]
I saw the lib render {children} 2 times like this:

```
<GestureHandler>
        <Animated.View ref={containerRef} style={containerStyle}>
          {children} => HERE
        </Animated.View>
      </GestureHandler>

      <Portal key={key} name={key}>
        <Animated.View
          key={key}
          style={portalContainerStyle}
          animatedProps={animatedPortalProps}
        >
          <PortalOverlay />
          {children} => HERE
        </Animated.View>
      </Portal>
```
So I think that make performance down to hell from here
